test/hotspot/jtreg/runtime/ErrorHandling/TestHeapDumpOnOutOfMemoryError.java

```diff
@@ -96,11 +97,28 @@ static void test(String type) throws Exception {
         output.stdoutShouldNotBeEmpty();
         output.shouldContain("Dumping heap to " + type + ".hprof");
         File dump = new File(heapdumpFilename);
-        Asserts.assertTrue(dump.exists() && dump.isFile(),
-                "Could not find dump file " + dump.getAbsolutePath());
+        Asserts.assertTrue(dump.exists() && dump.isFile(), "Could not find dump file " + dump.getAbsolutePath());
 
```

Comment:
I think you can remove this empty line, especially since you don't have one in the similar code below.